src/openms/thirdparty/hdf5-blosc-1.0.0/CMakeLists.txt

```diff
@@ -0,0 +1,88 @@
+cmake_minimum_required(VERSION 3.10)
+project(blosc_hdf5)
+include(ExternalProject)
+
+# options
+option(BUILD_TESTS
+    "Build test programs form the blosc filter" OFF)
+
+option(BUILD_PLUGIN
+    "Build dynamically loadable plugin for HDF5 version > 1.8.11" OFF)
+if(BUILD_PLUGIN)
+    set(PLUGIN_INSTALL_PATH "/usr/local/hdf5/lib/plugin" CACHE PATH 
+      "Where to install the dynamic HDF5-plugin")
+endif(BUILD_PLUGIN)
+
+set(BLOSC_PREFIX "${CMAKE_CURRENT_BINARY_DIR}/blosc")
+set(BLOSC_INSTALL_DIR "${CMAKE_CURRENT_BINARY_DIR}/blosc")
+set(BLOSC_CMAKE_ARGS -DCMAKE_INSTALL_PREFIX=${BLOSC_INSTALL_DIR})
+
+message("BLOSC_PREFIX='${BLOSC_PREFIX}'")
+message("BLOSC_INSTALL_DIR='${BLOSC_INSTALL_DIR}'")
+message("BLOSC_CMAKE_ARGS='${BLOSC_CMAKE_ARGS}'")
+message("GIT_EXECUTABLE='${GIT_EXECUTABLE}'")
+
+ExternalProject_Add(project_blosc
+  PREFIX ${BLOSC_PREFIX}
+  GIT_TAG v1.21.3
+  GIT_REPOSITORY https://github.com/Blosc/c-blosc.git    
+  INSTALL_DIR ${BLOSC_INSTALL_DIR}
+  CMAKE_ARGS ${BLOSC_CMAKE_ARGS}
```

Comment:
please build static

Reply:
static library or you man not from checkout?

Reply:
Static library

Reply:
You need to consult cblosc docs on how to build static